framework/meson.build

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 framework_config = configuration_data()
-framework_config.set10('SANDSTONE_STATIC', 0) # sysdeps may override
+framework_config.set10('SANDSTONE_STATIC', false) # sysdeps may override
```

Comment:
You beat me! 😁

Reply:
Oh-- missed the one in sysdeps/unix. Won't hold up merge for it-- we can snag it later.